### PR TITLE
TARIC validations

### PR DESCRIPTION
**Pull request by matthewford**

### CODE REVIEW

File: app/models/measure_type.rb [hunk 1]
```diff
@@ -11,13 +11,14 @@ class MeasureType < Sequel::Model
                         period_end_column:   :measure_types__validity_end_date
   plugin :oplog, primary_key: :measure_type_id
   plugin :conformance_validator
+  plugin :dirty
```
Comment (matthewford) [inline app/models/measure_type.rb]: duplicate if we are including this in all models

**Comment by matthewford** [comment]
We also have conflicts now @mauricioac need to rebase

**Comment by matthewford** [comment]
@macool even

**Comment by mauricioac** [comment]
This branch is not being used anymore.

**Pull request closed without merging**